Library/Formula/websocketd.rb

```diff
@@ -0,0 +1,24 @@
+class Websocketd < Formula
+  homepage "http://websocketd.com"
+  url "https://github.com/joewalnes/websocketd/archive/v0.2.10.tar.gz"
+  sha1 "4387c39aaf77e1bb5b6991358f02611a7b5dac1e"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+
+    mkdir_p "src/github.com/joewalnes"
+    ln_s "../../../", "src/github.com/joewalnes/websocketd"
+
+    system "go", "get", "./src/github.com/joewalnes/websocketd"
```

Comment:
That Makefile is mostly for our cross-platform build process. go_resource is definitely reasonable to use. Something like Library/Formula/rocket.rb looks close to what websocketd needs. There is single go library dependency, golang.org/x/net/websocket ;  not sure what else you'd like to hear